libcloud/compute/drivers/openstack.py

```diff
@@ -164,7 +164,8 @@ def list_nodes(self, ex_all_tenants=False):
         return self._to_nodes(
             self.connection.request('/servers/detail', params=params).object)
 
-    def create_volume(self, size, name, location=None, snapshot=None):
+    def create_volume(self, size, name, location=None, snapshot=None,
+                      volume_type=None):
```

Comment:
Sorry for the delay.

This is an extension argument (not part of the standard API) so it should be prefixed with `ex_` - `ex_volume_type`.